content/en/docs/distributions/nutanix/install-kubeflow.md

```diff
@@ -230,4 +235,13 @@ There are multiple ways to acces your Kubeflow Central Dashboard:
     kubectl -n istio-system apply -f certificate.yaml
     ```
   - You can now access the kubeflow dashboard by navigating to the istio-ingressgateway external IP e.g. `x.x.x.x`
-    
\ No newline at end of file
+
+
+## Access Grafana
+The default way to access the Grafana Dashboard is by using Port-Forward. You can port forward the servcie port to local port 8081.
+    
+   ```
+   kubectl --kubeconfig=./kaplin1.cfg port-forward svc/kubeflow-monitoring-grafana -n kubeflow-monitoring 8080:80
```

Comment:
Lets add 3 cases similar to central dashboard. Port-forward, NodePort and Load balancer way to access the grafana dashboard. also pls mention that this is only optional and will work if install_prometheus was set to true